examples/flux-test/zarf.yaml

```diff
@@ -29,3 +29,20 @@ components:
       - https://github.com/stefanprodan/podinfo.git
     images:
       - ghcr.io/stefanprodan/podinfo:6.3.3
+    actions:
+      onDeploy:
+        after:
+          # Waiting using the wait-for command
+          - description: "Podinfo deployment to be available via cmd action"
+            cmd: "./zarf tools wait-for deployment podinfo available -n podinfo"
+            mute: true
```

Comment:
Do we want to leave this in here? I think we should push people to use `wait` in yaml rather than this command (even though they _can_)

Reply:
Yeah, that was a holdover to test the path before I decided to use the actual wait-for command, so now it isn't needed.